packages/caravan-psbt/src/psbtv0/psbt.ts

```diff
@@ -349,6 +349,7 @@ inputs to protect against large fee attack`);
       index: input.index,
       transactionHex: fundingTxHex,
       txid: reverseBuffer(input.hash).toString("hex"),
+      sequence: input.sequence,
```

Comment:
this is required for the trezor interaction which needs to have inputs converted in a way that its api will work with so the input sequence needs to be passed through as well.